[mri violations] MRI Violations module does not describe or test widgets (#6458)
aces/Loris: modules/mri_violations/README.md

"Protocol Violation"
- `MRICandidateErrors` logs all the scans referring to the problem
"Candidate Mismatch"

## Dashboard Widget

The total `Violated scans` will be shown inside the Dashboard widget, named "My Tasks"

> and when clicked will link to the mri_violations module.

Reviewer: ```suggestion
with a link pointing to the mri_violations module.
```
